React/tic-tac/src/app/screens/Login/components/Form.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import validate from '../components/formField/validate';
+
+import style from './styles.scss';
+import renderField from './formField/index';
+
+const loginForm = props => {
+  const { handleSubmit, onSubmit, submitting, onClick } = props;
+
+  return (
+    <form
+      onSubmit={handleSubmit(onSubmit)}
+      className={`${style.login} ${style.fadeInDown} ${style.animated}`}
+    >
+      <Field name="email" type="email" component={renderField} label="Email" className={style.inputField} />
+      <Field
+        name="password"
+        type="password"
+        component={renderField}
+        label="Password"
+        className={style.inputField}
+      />
+      <div>
+        <button
+          type="submit"
+          disabled={submitting}
+          className={`${style.inputField} ${style.submit}`}
+          onClick={onClick}
+        >
+          Submit
+        </button>
+      </div>
+    </form>
+  );
+};
+
+loginForm.propTypes = {
+  handleSubmit: PropTypes.func,
+  onSubmit: PropTypes.func.isRequired,
+  submitting: PropTypes.bool,
+  onClick: PropTypes.func
+};
+
+export default reduxForm({
+  form: 'syncValidation',
```

Comment:
Extract this form name as a const in `src/contstants/formNames.js` Some forms can have many screens and in each screen must have the same name, or also maybe you need to refer to the form with the name. For all this things, u need one name, and if u want change this name or refer this form, is more easier to do that from one file.